src/utils/orgUtils.ts

```diff
@@ -65,6 +72,103 @@ export class OrgUtils {
         }
     }
 
+    public static async getAllCompactLayoutsForSObject(
```

Comment:
Nit: I'd just call this `getCompactLayoutsForSObject`. The "all" is pretty much implied.